### DISCUSSION TITLE
How to install the PKG package for all users in a company using Terminal

### DISCUSSION THREAD

**Discussion by eubrendalau** [Homebrew/discussions#2416] [Getting started]
Hi, I'm new to scripting
I need to write a script where the command is: install the PKG package for all users in my company, using Terminal
At first I'm using the following command to download the PKG: sudo installer -pkg/Users/<your_account>/downloads/package.pkg/ -target /Users/<your_account>/Applications
Is it possible to download the PKG file from the internet, by command, from a link? (This before running the install command)
And how do I install them all?
What would this script look like?
Help!!!!

**Comment by ipatch** [comment]
not sure how this homebrew related 🤨. but something i have noticed and probably you as well. that certain `.pkg` files ie. (thinking off the top of my head) virtualbox (comes to mind) that it prompts the user who is running the package installer whether to install the package for the specific user or for all users on the sytem. so if i had to guess,

- the package you are attempting to install, may require being setup to have option to install for all users on the system, because i believe certain packages do not prompt to install for a single user or for all users.
